### PR TITLE
Fixing links and various changes

### PR DESCRIPTION
**Pull request by DCtheGeek**
Fixing broken links based on build notifications.  Cleaned up various other items at the same time (lint, powershell tags, broken code, code normalization, etc.).

### CODE REVIEW

File: reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md [hunk 1]
```diff
@@ -67,6 +67,5 @@ see [How to Save a Script](How-to-Write-and-Run-Scripts-in-the-Windows-PowerShel
 
 ## See Also
 
-- [Using the Windows PowerShell ISE](Using-the-Windows-PowerShell-ISE.md)
-- [How to Use the Console Pane in the Windows PowerShell ISE](How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md)
-
+- [Introducing the Windows PowerShell ISE](Introducing the Windows PowerShell ISE)
```
Comment (sdwheeler) [inline reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md]: The URL portion of this link is invalid.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fcore-powershell%2Fise%2FHow-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md)
[reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fsetup%2FAccessibility-in-Windows-PowerShell-ISE.md)
[reference/docs-conceptual/core-powershell/ise/How-to-Use-Profiles-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-Profiles-in-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-Tab-Completion-in-the-Script-Pane-and-Console-Pane.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-Tab-Completion-in-the-Script-Pane-and-Console-Pane.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISE-Object-Model-Hierarchy.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISE-Object-Model-Hierarchy.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEAddOnTool-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEAddOnTool-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEAddOnToolCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEAddOnToolCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEEditor-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEEditor-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEFile-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEFile-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEFileCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEFileCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEMenuItem-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEMenuItem-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEMenuItemCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEMenuItemCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEOptions-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEOptions-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISESnippetCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISESnippetCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ObjectModelRoot-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ObjectModelRoot-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-PowerShellTab-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-PowerShellTab-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-PowerShellTabCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-PowerShellTabCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/getting-started/cookbooks/Other-Useful-Scripting-Objects.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/getting-started/cookbooks/Other-Useful-Scripting-Objects.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/getting-started/fundamental/Exploring-the-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/getting-started/fundamental/Exploring-the-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |

<a name="reference%2Fdocs-conceptual%2Fcore-powershell%2Fise%2FHow-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md"></a>
### [reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md)
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE).

<a name="reference%2Fdocs-conceptual%2Fsetup%2FAccessibility-in-Windows-PowerShell-ISE.md"></a>
### [reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md)
 - **Line 156**: **[Warning]** Invalid file link:(~/core-powershell/ise/Introducing-the-Windows-PowerShell-ISE.md).
 - **Line 156**: **[Warning]** Invalid file link:(~/core-powershell/ise/Introducing-the-Windows-PowerShell-ISE.md).
 - **Line 156**: **[Warning]** Invalid file link:(~/core-powershell/ise/Introducing-the-Windows-PowerShell-ISE.md).
 - **Line 156**: **[Warning]** Invalid file link:(~/core-powershell/ise/Introducing-the-Windows-PowerShell-ISE.md).
 - **Line 156**: **[Warning]** Invalid file link:(~/core-powershell/ise/Introducing-the-Windows-PowerShell-ISE.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C15%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201803152118163204-2181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=neBNamL%2B6TxCopNDf80%2B0l0HFev6VhEgirX5v8hTj7U%3D&st=2018-03-15T21%3A41%3A06Z&se=2018-04-15T21%3A46%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md) | :warning:Warning | | [Details](#user-content-reference%2Fdocs-conceptual%2Fcore-powershell%2Fise%2FHow-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md)
[reference/docs-conceptual/core-powershell/ise/How-to-Use-Profiles-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-Profiles-in-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-Tab-Completion-in-the-Script-Pane-and-Console-Pane.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-Tab-Completion-in-the-Script-Pane-and-Console-Pane.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISE-Object-Model-Hierarchy.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISE-Object-Model-Hierarchy.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEAddOnTool-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEAddOnTool-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEAddOnToolCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEAddOnToolCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEEditor-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEEditor-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEFile-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEFile-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEFileCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEFileCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEMenuItem-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEMenuItem-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEMenuItemCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEMenuItemCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEOptions-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEOptions-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISESnippetCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISESnippetCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ObjectModelRoot-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ObjectModelRoot-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-PowerShellTab-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-PowerShellTab-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-PowerShellTabCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-PowerShellTabCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/getting-started/cookbooks/Other-Useful-Scripting-Objects.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/getting-started/cookbooks/Other-Useful-Scripting-Objects.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/getting-started/fundamental/Exploring-the-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/getting-started/fundamental/Exploring-the-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |

<a name="reference%2Fdocs-conceptual%2Fcore-powershell%2Fise%2FHow-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md"></a>
### [reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md)
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE.md).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE.md).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE.md).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE.md).
 - **Line 70**: **[Warning]** Invalid file link:(~/docs-conceptual/core-powershell/ise/Introducing the Windows PowerShell ISE.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C15%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201803152156309053-2181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=oFRaQlYkWcfXG7b4q%2F4M8B7nWEK2Jziy%2F%2BuqOL%2BC4l8%3D&st=2018-03-15T22%3A19%3A09Z&se=2018-04-15T22%3A24%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Create-a-PowerShell-Tab-in-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-Profiles-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-Profiles-in-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-Tab-Completion-in-the-Script-Pane-and-Console-Pane.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-Tab-Completion-in-the-Script-Pane-and-Console-Pane.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/How-to-Use-the-Console-Pane-in-the-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISE-Object-Model-Hierarchy.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISE-Object-Model-Hierarchy.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEAddOnTool-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEAddOnTool-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEAddOnToolCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEAddOnToolCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEEditor-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEEditor-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEFile-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEFile-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEFileCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEFileCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEMenuItem-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEMenuItem-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEMenuItemCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEMenuItemCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISEOptions-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISEOptions-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ISESnippetCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ISESnippetCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-ObjectModelRoot-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-ObjectModelRoot-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-PowerShellTab-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-PowerShellTab-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/core-powershell/ise/The-PowerShellTabCollection-Object.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/core-powershell/ise/The-PowerShellTabCollection-Object.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/getting-started/cookbooks/Other-Useful-Scripting-Objects.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/getting-started/cookbooks/Other-Useful-Scripting-Objects.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/getting-started/fundamental/Exploring-the-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/getting-started/fundamental/Exploring-the-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |
[reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md](https://github.com/DCtheGeek/PowerShell-Docs/blob/dmc-fixlinks/reference/docs-conceptual/setup/Accessibility-in-Windows-PowerShell-ISE.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C15%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201803152159177498-2181%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RXgQ5uke8j%2F5%2BjGHvnT05fZLAwuPX3fHV6qtBTI47NI%3D&st=2018-03-15T22%3A50%3A09Z&se=2018-04-15T22%3A55%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**